src/types/events.ts

```diff
@@ -59,3 +60,143 @@ export class BalancesTransferEvent {
         return this._chain.decodeEvent(this.event)
     }
 }
+
```

Comment:
The bifrost sample contains A LOT more events, this has been trimmed and only contains the ones that seem to be necessary for the current iteration. We can add as necessary all the others